lua/wire/client/e2descriptions.lua

```diff
@@ -313,12 +313,12 @@ E2Helper.Descriptions["setTrails(e:nnnsvn)"] = "StartSize, EndSize, Length, Mate
 E2Helper.Descriptions["setTrails(e:nnnsvnnn)"] = "StartSize, EndSize, Length, Material, Color (RGB), Alpha, AttachmentID, Additive. Adds a trail to E with the specified attributes"
 E2Helper.Descriptions["removeTrails(e:)"] = "Removes the trail from E"
 E2Helper.Descriptions["runOnKeys(en)"] = "If set to 1, E2 will run when specified player presses/releases their key"
-E2Helper.Descriptions["disconnectClk()"] = "Returns 1 if the chip is being executed because of a disconnect event. Returns 0 otherwise"
-E2Helper.Descriptions["lastDisconnected()"] = "Returns the last player to disconnect. Must be done while in a disconnectClk() as anytime after the player object is gone."
-E2Helper.Descriptions["runOnDisconnect(n)"] = "If set to 0, the chip will no longer run on disconnect events, otherwise it makes this chip execute when someone disconnects. Only needs to be called once, not in every execution"
-E2Helper.Descriptions["connectClk()"] = "Returns 1 if the chip is being executed because of a connect event. Returns 0 otherwise"
-E2Helper.Descriptions["lastConnected()"] = "Returns the last player to connect."
-E2Helper.Descriptions["runOnConnect(n)"] = "If set to 0, the chip will no longer run on connect events, otherwise it makes this chip execute when someone connects. Only needs to be called once, not in every execution"
+E2Helper.Descriptions["disconnectedPlayerClk()"] = "Returns 1 if the chip is being executed because of a player disconnect event. Returns 0 otherwise"
+E2Helper.Descriptions["lastDisconnectedPlayer()"] = "Returns the last player to disconnect. Must be done while in a disconnectClk() as anytime after the player object is gone."
+E2Helper.Descriptions["runOnPlayerDisconnect(n)"] = "If set to 0, the chip will no longer run on player disconnect events, otherwise it makes this chip execute when someone disconnects. Only needs to be called once, not in every execution"
+E2Helper.Descriptions["connectedPlayerClk()"] = "Returns 1 if the chip is being executed because of a player connect event. Returns 0 otherwise"
```

Comment:
Out of all these grammar changes, I do agree with this one.
`playerConnectClk` Makes sense, is lower character count and flows better in my opinion.
And of course if it were to be changed to that the others should reflect the same format.